src/vm/prestub.cpp

```diff
@@ -49,6 +49,10 @@
 #include "stacksampler.h"
 #endif
 
+#ifdef FEATURE_PERFMAP
```

Comment:
@mmitche, I've actually removed the ifdefs from perfmap.h and left these in.  I had originally not had ifdefs in perfmap.h, but added them when I was trying to fix the windows build issue I was having.